src/site/content/en/lighthouse-accessibility/aria-valid-attr-value/index.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: post
+title: Ensure ARIA attributes have valid values
+description: |
+  Learn about aria-valid-attr-value audit.
+author: megginkearney
+web_lighthouse:
+  - aria-valid-attr-value
+---
+
+[ARIA](https://www.w3.org/TR/wai-aria-1.1/#role_definitions)
+roles and attributes help screen readers
+provide missing information about an element.
+For these roles and attributes to make sense,
+each ARIA `role` supports a specific subset of `aria-*` attributes
+(see [ARIA roles definitions](https://www.w3.org/TR/wai-aria-1.1/#role_definitions)).
+Assistive technologies, like screen readers,
+can't interpret ARIA attributes with invalid values.
+Lighthouse reports when attributes do not have valid values:
+
+<figure class="w-figure">
+  <img class="w-screenshot w-screenshot--filled" src="aria-valid-attr-value.png" alt="Lighthouse: ARIA attributes do not have valid values">
```

Comment:
just noticed the alt text matches the figcaption. I think the figcaptions are perfect as they are, but the alt text should describe the image. In this case a good alt might be, "Lighthouse audit showing aria-checked without the value 'true'".

Reply:
Cool. I'll improve the alt text across all the accessibility guides now for this set of changes. I'm also adding a todo to the bigger list of todos to sweep all audit alt text to make it better too!